shared/elements/gaia-toast/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
This page shouldn't be committed. I've [update the bower.json](https://github.com/gaia-components/gaia-toast/commit/09392f7d2f130c9b61418abef7a450817a1628ae). Please `bower update gaia-toast` to pull `HEAD` into Gaia.